{"name": "test_register_failure_001", "status": "failed", "statusDetails": {"message": "AssertionError: 405 != 400", "trace": "self = <test_Newregister.TestNewregister testMethod=test_register_failure_001>\n\n      def test_register_failure_001(self):\n          response = requests.get('http://127.0.0.1:8000/home/api/register/', params= {\n    \"email\": \"user.four1@example.com\",\n    \"password\": \"anotherPass124\"})\n>         self.assertEqual(response.status_code, 400)\nE         AssertionError: 405 != 400\n\ntestplatform\\tests\\unit\\test_Newregister.py:19: AssertionError"}, "start": 1743386552896, "stop": 1743386552915, "uuid": "e9f33d80-5ddf-403e-9cb3-6af58daf6c28", "historyId": "347ed02d9742c71ff7ee7013be9b6ceb", "testCaseId": "347ed02d9742c71ff7ee7013be9b6ceb", "fullName": "testplatform.tests.unit.test_Newregister.TestNewregister#test_register_failure_001", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Newregister"}, {"name": "subSuite", "value": "TestNewregister"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "22000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Newregister"}]}